_includes/ato_grid.html

```diff
@@ -31,8 +31,8 @@ <h2 class="mdl-card__title-text">{{ automation.title }}</h2>
           <!-- Automation Readme url is located in links index 1 -->
           <!-- We will need to design a README.md button -->
           <a href="{{ automation.links[1].url }}"
-            class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect">
-            Read Me
+            class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" style="font-family: Poppins">
```

Comment:
Is there a reason that you used inline styles here rather than creating a class?  In general, using inline styles is not considered a best practice. 

Here's someone writing way more about it that is probably necessary: https://www.lifewire.com/avoid-inline-styles-for-css-3466846

Reply:
@drubgrubby  These styles don't display unless they are inline styles. @mcspach  used a trick to bypass inline styling that I am sure we could implement here as well.